samtranslator/translator/transform.py

```diff
@@ -15,13 +19,19 @@ def transform(input_fragment, parameter_values, managed_policy_loader, feature_t
     sam_parser = Parser()
     to_py27_compatible_template(input_fragment, parameter_values)
     translator = Translator(  # type: ignore[no-untyped-call]
-        managed_policy_loader.load(),
+        None,
         sam_parser,
     )
+
+    @lru_cache(maxsize=None)
```

Comment:
Nit/question: Why `maxsize=None` vs `mazsize=1`?

Reply:
One fewer things to think about and avoid bugs if we e.g. add an optional parameter later.

`@lru_cache(maxsize=None)` is same as `@cache` in Python 3.9+, i.e. a simple args-to-value cache without the LRU machinery.